packages/utils-helpers/src/svgToDataUri.ts

```diff
@@ -0,0 +1,47 @@
+const REGEX = {
```

Comment:
lol 😅 i hope you dindt had to write this yourself? :D I think more simple it would be to just use base64 decode and done :D

Edit: see comment above - i believe we dont need this at all hopefully

Reply:
Can discuss other solutions of course :)

Reply:
I simplified that and now use base64 for the inline dataUri SVG stuff.